README.md

```diff
@@ -1 +1,154 @@
-# WebServ
+# WebServ: JAS Server
+
+A simple HTTP server written in C++
+
+This project introduced us to the wonderful world of HTTP, RFC, server configurations, CGI, and TCP socket mangement. 
+This simple server provides GET, HEAD, POST, and DELETE methods and provides common gateway interface (CGI) for scripted/  interpred languages such as python and bash. It was developed to specifically work with a chrome client and run on Mac OS 10.14.6.
+
+## Collaborators
+
+Team JAS!
+
+[Jenny Boucher](https://github.com/jboucher154)
+
+[Azär Sarikhani](https://github.com/azarSarikhani)
+
+[Sakari Salmi](https://github.com/sakarisalmi)
+
+## Project Coding Standards
+
+Coding standards set for the project can be found [here](project_standards/standards.md)
+
+The project began with guidelines to use C++98 standard. The limit was lifted towoards the end of the project so no major refactoring was done, yet a few methods and objects from standards beyond 98 can be found.
```

Comment:
"to use with c++98 standard", towards, so -> but, beyond c++98

Reply:
I rewrote it with grammarly.